### PR TITLE
Added some content, commented out few navbar items and changed the banner image

### PR DESCRIPTION
**Pull request by duttakapil**
Demo : http://kd-fcc-hyd.surge.sh/

@vinaypuppal can you review this and then merge

### CODE REVIEW

**Comment by vinaypuppal** [comment]
@duttakapil it would be better if u include a demo site as well in PR. You can use surge.sh to do that.

File: index.html [hunk 1]
```diff
@@ -22,47 +22,47 @@
...
+          <!-- <li class="header__navMenuItem">
             <a class="header__navMenuLink" href="/jobs">Jobs</a>
-          </li>
-          <li class="header__navMenuItem">
+          </li> -->
+          <!-- <li class="header__navMenuItem">
             <a class="header__navMenuLink" href="/shop">Shop</a>
-          </li>
+          </li> -->
         </ul>
```
Comment (vinaypuppal) [inline index.html]: why did u comment them??

File: css/header.css [hunk 2]
```diff
@@ -1,10 +1,10 @@
 .header {
   background: green;
-  background-image: url(https://fcc-hyd.github.io/img/footer_lodyas.png);
+  background-image: url(../media/meetup.jpg);
   background-size: cover;
   background-position: center;
   background-repeat: no-repeat;
-  height: 500px;
+  height: 100vh;
```
Comment (vinaypuppal) [inline css/header.css]: until we add countdown timer keeping it 500px would be good i think

File: index.html (hunk 1 above)
Comment (duttakapil) [inline index.html]: Yes. The about and the meetups section for sufficient for now isley

File: css/header.css (hunk 2 above)
Comment (duttakapil) [inline css/header.css]: We'll be adding the timer pretty soon anyway, so why not keep it

File: index.html (hunk 1 above)
Comment (vinaypuppal) [inline index.html]: but navbar looks odd with just two elements, i think showing 404(as github shows) or using old links would be better.

Comment (duttakapil) [inline index.html]: Uncommented the ones that were working before. Updated the demo as well : http://kd-fcc-hyd.surge.sh/

**Pull request merged**